python/tvm/relay/op/_transform.py

```diff
@@ -63,6 +63,8 @@
 _reg.register_injective_schedule("sparse_to_dense")
 _reg.register_injective_schedule("matrix_set_diag")
 _reg.register_injective_schedule("adv_index")
+_reg.register_injective_schedule("sparsereshape")
```

Comment:
I think `sparse_reshape` would be a more appropriate name given the above naming conventions.

Reply:
Done.